model/DataObject.php

```diff
@@ -1959,7 +1959,9 @@ public function db($fieldName = null) {
 				$items = isset($items) ? array_merge((array) $items, $dbItems) : $dbItems;
 			}
 		}
-
+		if ($fieldName) {
```

Comment:
Maybe add a comment here on *why* its returning? I had to read through the code twice to figure out that somewhere above it, there's a deeply nested `return` for this case heh

Reply:
ok.